.github/scripts/run-sweep-benchmark.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -xeo pipefail
+
+for CONFIG in /output/configs/*; do
```

Comment:
where does /output/configs come from? and what what is the format of CONFIG?

Reply:
/output/configs is defined in as a list of "requirements.txt" files, e.g. "requirements-1.6.0.dev20200526.txt". Inside this txt file is the urls of torch/torchtext/torchvision wheels, for example, "requirements-1.6.0.dev20200526.txt" contains:
https://download.pytorch.org/whl/nightly/cu102/torch-1.6.0.dev20200526-cp37-cp37m-linux_x86_64.whl
https://download.pytorch.org/whl/nightly/cu102/torchvision-0.7.0.dev20200526-cp37-cp37m-linux_x86_64.whl
https://download.pytorch.org/whl/nightly/cu102/../torchtext-0.6.0.dev20200526-cp37-cp37m-linux_x86_64.whl